iree/tools/BUILD

```diff
@@ -139,13 +139,26 @@ cc_library(
     ],
 )
 
+cc_library(
+    name = "init_xla_dialects",
+    hdrs = [
+        "init_xla_dialects.h",
+    ],
+    deps = [
+        "@llvm-project//mlir:IR",
+        "@org_tensorflow//tensorflow/compiler/mlir/xla:hlo",
+        "@org_tensorflow//tensorflow/compiler/mlir/xla:lhlo",
+    ],
+)
+
 cc_library(
     name = "iree_opt_main",
     srcs = ["opt_main.cc"],
     deps = [
         ":init_compiler_modules",
         ":init_passes_and_dialects",
         ":init_targets",
+        ":init_xla_dialects",
```

Comment:
We should be able to drop `xla_dialect_registration` below. If we can't, then I think this didn't work properly :-P

Reply:
Giving the CI a try :-)

Reply:
Looks like it worked :-)